### ISSUE TITLE
Coreograph : need to flag an error when no cores found

### ISSUE THREAD

**Issue by clarenceyapp** [labsyspharm/mcmicro#71]
When cores are significantly different from the training data, the model may not be able to detect any cores and returns a very dim probability map. This in turn will segment nothing. But by this time, dearraying and UnMicst would have appeared to have completed. Need a check statement.

**Comment by clarenceyapp** [comment]
@ArtemSokolov how might nextflow be alerted by coreograph when something has gone wrong and that the user needs to retry with different parameters? Posting an error message is informative but won't stop nextflow from proceeding.

**Comment by ArtemSokolov** [comment]
Can MATLAB throw an exception that causes the `matlab -r` command to exit with a non-zero status? Nextflow only proceeds if a process exits with zero (success). Otherwise, it terminates with something like this:
```
Error executing process > 'abc'

Caused by:
  Process `abc` terminated with an error exit status (1)

Command executed:
  python abc.py

Command exit status:
  1

Command output:
  (empty)

Command error:
  Traceback (most recent call last):
    File "abc.py", line 1, in <module>
      import somepackage
  ModuleNotFoundError: No module named 'somepackage'
```

You can use an exception to communicate your message, which will then appear in `command error` when Nextflow terminates.

**Comment by clarenceyapp** [comment]
For my reference: Matlab can send an exit code between 0-255 using 'quit'

**Comment by clarenceyapp** [comment]
This commit will exit Matlab, print an error message, and give exit code 255 if there are no detected cores or less than 50% of the cores were actually outlined. Tested locally.
c4bd1432c3e01771f08ef6fed99d6bf5a98d37ef

**Comment by clarenceyapp** [comment]
@ArtemSokolov need another parameter when calling Coreograph called `'cluster' , true` when using nextflow on line 119 of main.nf
'    "addpath(genpath('${params.tool_core}')); \ 
    tmaDearray('./$s','outputPath','.','useGrid','false','cluster',true); exit" '

**Comment by ArtemSokolov** [comment]
Addressed in #87
